components/SecondaryButton.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<button class="secondary-button" @click="handleClick"><slot></slot></button>
+</template>
+
+<script>
+export default {
+	methods: {
+		handleClick() {
+	      this.$emit("buttonClicked", {});
+	  	}
+	}
+}
+</script>
+
+<style scoped lang="scss">
+ .secondary-button {
+ 	font-family: $headline-font;
+ 	font-size: $f-4;
+ 	box-sizing: border-box;
+ 	color: $white-color;
+	padding: $offset-2;
```

Comment:
this indentation seems off.
please run 
```
yarn lintfix
```
and watch the auto formatting magic happen, at least before you merge. this helps to make everything consistent, lowers our human effort spent on formatting and prevents errors.

I also recommend setting up your text editor to automatically run prettier on save. it feels weird at first, but you will love it very soon!